metadata-generator/src/main/java/com/splunk/opentelemetry/tools/MetadataGenerator.java

```diff
@@ -3626,15 +3630,15 @@ Some instrumentations (e.g. Spring Web MVC instrumentation) produce SpanKind.Int
     instrumentations.add(
         splunkInstrumentation("jvm-metrics.splunk")
             .component("Java Platform", null)
-            .metric(
+            .customMetric(
```

Comment:
@breedx-splk it seems like memory profiling metrics are currently custom

Reply:
I have no idea, but that's fine. Classification shouldn't be an instrumentation concern anyway.